properties/base/instance.py

```diff
@@ -101,8 +101,14 @@ def validate(self, instance, value):
             if isinstance(value, dict):
                 return self.instance_class(**value)
             return self.instance_class(value)
-        except (ValueError, KeyError, TypeError):
-            self.error(instance, value)
+        except GENERIC_ERRORS as err:
+            if hasattr(err, 'error_tuples'):
+                extra = '({})'.format(' & '.join(
```

Comment:
this is strange.

Reply:
Yeah, I agree with this. We already are keeping some error info in a structured object; totally makes sense to keep it all structured.